### ISSUE TITLE
Adjust all files that have the EJC name or number in it

### ISSUE THREAD

**Issue by inbaz** [inbaz/ers#697]
https://github.com/inbaz/ers/pull/696 In this pull request all files that contain 2015 and 38th were changed. Adjust these files to use the configuration settings of ers.local.php

**Comment by jolieno** [comment]
Oscar didn't you do this yesterday?

**Comment by odegroot** [comment]
No, I replaced 100+ occurences of 2015 / 38th. The scope of this issue is replacing all of those occurences with a *reference* to the value set in `ers.local.php`, so that from that point onward, you can just update that single file instead of updating the same string in dozens of different files.

In other words, what I did was the quick fix for this year, whereas this issue provides the real solution for next years.
